apps/api/src/modules/stores/stores.service.spec.ts

```diff
@@ -1,20 +1,113 @@
+import { CreateStoreDto, UpdateStoreDto } from '@meal-time/api-interfaces';
 import { ConfigService } from '@nestjs/config';
 import { Test, TestingModule } from '@nestjs/testing';
+import { Stores } from '@prisma/client';
 import { PrismaService } from '../../services/prisma/prisma.service';
 import { StoresService } from './stores.service';
 
 describe('StoresService', () => {
-  let service: StoresService;
+	let service: StoresService;
+	let prisma: PrismaService;
+	let store1: Stores;
 
-  beforeEach(async () => {
-    const module: TestingModule = await Test.createTestingModule({
-      providers: [StoresService, PrismaService, ConfigService],
-    }).compile();
+	beforeEach(async () => {
+		const module: TestingModule = await Test.createTestingModule({
+			providers: [StoresService, PrismaService, ConfigService],
+		}).compile();
 
-    service = module.get<StoresService>(StoresService);
-  });
+		service = module.get<StoresService>(StoresService);
+		prisma = module.get<PrismaService>(PrismaService);
+		await prisma.cleanDb();
 
-  it('should be defined', () => {
-    expect(service).toBeDefined();
-  });
+		const dto: CreateStoreDto = {
+			name: 'Brugsen',
+		};
+
+		const dto1: CreateStoreDto = {
+			name: 'Fakta',
+		};
+
+		store1 = await prisma.stores.create({
+			data: {
+				...dto,
+			},
+		});
+
+		await prisma.stores.create({
+			data: {
+				...dto1,
+			},
+		});
+	});
+
+	it('should create store', async () => {
+		const dto: CreateStoreDto = {
+			name: 'Meny',
+		};
+		const store = await service.create(dto);
+
+		return expect(store).toHaveProperty('name');
+	});
+
+	it('should find all stores', async () => {
+		const prismaStores = await prisma.stores.findMany();
+		prismaStores.sort((a, b) => {
+			if (a.id > b.id) {
+				return 1;
+			} else if (a.id < b.id) {
+				return -1;
+			}
+			return 0;
+		});
+
+		const stores = await service.findAll();
+
+		return expect(stores).toEqual(prismaStores);
+	});
+
+	it('should find specific store', async () => {
+		const foundStore = await service.findOne(store1.id);
+
+		return expect(foundStore).toEqual(store1);
+	});
```

Comment:
I would probably get rid of `store1` all together and always create the store you need to check in each test. This makes it easier when looking at the test to understand what the test is doing.